packages/frontend/src/components/NavBar/DashboardExplorerBanner.tsx

```diff
@@ -41,7 +41,7 @@ export const DashboardExplorerBanner: FC<Props> = ({ projectUuid }) => {
                 return 'Return to dashboard';
             case 'creating':
             case 'editing':
-                return 'Cancel';
+                return 'Return to dashboard';
```

Comment:
We are missing a bit of logic as we only want this if there are no unsaved changes.

eg:

1. page load = Return to dashboard
2. I add a new metric = Cancel
3. I click save = Return to dashboard
4. I change the chart config = Cancel
5. I save = Return to dashboard